devenv/dev-dashboards/panel-candlestick/candlestick.json

```diff
@@ -21,10 +24,15 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 2,
+  "id": 93,
   "links": [],
   "liveNow": false,
   "panels": [
     {
+      "datasource": {
+        "type": "testdata",
+        "uid": "3fuUBUNGz"
```

Comment:
@sdboyer 
Where did this come from? It is not the default data source, the uid is wrong

Reply:
sorry, IIRC i just did an export via the UI and dumped it to disk - that's what popped out